src/WebJobs.Script/Description/DotNet/Compilation/Raw/RawAssemblyCompilation.cs

```diff
@@ -52,7 +52,7 @@ public FunctionSignature GetEntryPointSignature(IFunctionEntryPointResolver entr
                 throw new InvalidOperationException($"The function type name '{typeName}' is invalid.");
             }
 
-            MethodInfo method = functionType.GetMethod(methodName, BindingFlags.Static | BindingFlags.Public);
+            MethodInfo method = functionType.GetMethod(methodName, BindingFlags.Static | BindingFlags.Instance | BindingFlags.Public);
```

Comment:
I couldn't find any tests for this code, so I don't have a suite to add a new test to. Is there an existing suite? If not, I'll create a tracking item to get tests added for this code.